src/modules/file.ts

```diff
@@ -16,6 +29,38 @@ function extractHeaders (options?: OptionsUpload): Dictionary<boolean | number>
   return headers
 }
 
+function readContentDispositionFilename (header?: string): string {
+  try {
+    if (header == null) {
+      throw new BeeError('missing content-disposition header')
+    }
+    const disposition = contentDisposition.parse(header)
+
+    if (disposition?.parameters?.filename != null) {
```

Comment:
```suggestion
    if (disposition?.parameters?.filename) {
```

The weakly typed test for is unnecessary and in fact it will never be `null` just `undefined` (see https://github.com/jshttp/content-disposition/blob/master/index.js#L353)

Reply:
That's true. Out of habit I usually use the `== null` whenever I want to make sure something is null _or_ undefined when using optional chaining (instead of any truthy value), so to me it's natural to write this when checking for the inverse. Nevertheless I will change it.